src/actions/userActions.js

```diff
@@ -1,13 +1,12 @@
 import ActionTypes from './actionTypes';
 import ActionUtil from './actionUtil';
+import {getRoaster} from './roasterActions';
 
-const AUTHENTICATE_USER_URL = 'https://towncenter.expresso.store/api/user/login';
-const USER_URL = 'https://towncenter.expresso.store/api/user/';
+const AUTHENTICATE_USER_URL = 'https://towncenter.expresso.store/api/auth/login';
```

Comment:
This reflects a change on the back end of TownCenter so that we can use tokens to get users but still have clear urls.